### PR TITLE
Add a blog: 女生科技体验节，Wechaty杂货铺

### PR DESCRIPTION
**Pull request by TingYinHelen**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [ ] Add Blog Header, including title, author, date.
e.g.
```
---
 layout: post
 title: "'Score Your Face Photo' a ML&Wechaty practice"
 date: 2017-09-18 09:00 +0800
 author: huyingxi
 ---
 
 > Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```
- [ ] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [ ] Embed the photo & video before publishing, save all external file to the blog `/download/2018` directory.
- [ ] Add `<!--more-->` section to just show the abstract in blog homepage.

[See more](https://github.com/Chatie/blog#contribute-guideline)

### CODE REVIEW

File: _posts/2018-02-06-techie-festival-FAQ.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+这篇文章主要针对女生科技体验节Wechaty专场，同学们问的问题的汇总。因为很多同学没有太多编程基础，所以这篇文章讲解会比较基础。
+
+故事从哪里说起呢？通过这次女生科技体验节活动，同学们遇到的问题主要来自三方面，第一，是docker安装方面的问题；第二，是对一些专有名词的不理解；第三，是程序运行和代码编写方面的问题。那么，我会把重心放在，程序编写和运行中遇到的问题，和对同学们来说比较陌生的专业术语的解释上。因为，安装的问题五花八门，安装好以后大家都就可以随心所欲的写自己的机器人了，遇到更多的是程序代码上的问题。还有，我对安装真的不是特别在行。。。。
+
+So，这篇文章主要以问答的形式来写，并且，后面遇到新的问题，也可以告诉我，我会及时的补充。
+
+#### 1.  Wechaty微信机器人可以用到公众号上吗？
+
+不能用于公众号，公众号本身有自动回复的功能，可以直接调用公众号的接口，具体可以查看文档，使用Wechaty制作微信机器人只能用在个人微信号上。
```
Comment (lijiarui) [inline _posts/2018-02-06-techie-festival-FAQ.md]: 加上下面的内容：

但是wechaty 有计划把公众号整合进来，详细讨论见 [issue1016](https://github.com/Chatie/wechaty/issues/1016)

File: _posts/2018-02-06-techie-festival-FAQ.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+当接收到的消息是自己发出的，就直接终止代码继续往后执行。
+
+#### 4.  Windows7下发送不了中文message
+
+这是MinGW的问题。MinGW 是一组包含文件和端口库，其功能是允许控制台模式的程序使用微软的标准C运行时（C Runtime）库。
+简单的说，有一些软件必须在Linux的环境下运行，MinGW可以在Windows上提供一套Linux的运行的环境，表象的说就是可以在Windows上执行Linux的命令。但是MinGW对中文的支持很不友好。  我们在安装git的时候会自动安装MinGW。所以导致同学们发送不了中文的message。
+具体的解决方法就是：使用命令`docker-machine ssh`，ssh连接到docker镜像内部，然后再运行。ssh是一种网络协议。用户想要通过本地计算机登录到远程计算机，就要使用ssh。具体想要多了解的可以看阮一峰的这篇小白入门文章：[http://www.ruanyifeng.com/blog/2011/12/ssh_remote_login.html](http://www.ruanyifeng.com/blog/2011/12/ssh_remote_login.html) 。这里我们使用docker-machine ssh访问到docker镜像，因为docker内部就是linux环境，所以内部没有用到MinGW，这样就解决了系统的兼容性问题。
+但是ssh进入内部，就不能直接使用docker run -t挂载本地目录，因为现在命令的执行是直接在虚拟机内部的，操作不了宿主机的文件。解决的方法是，把本地的目录挂在到docer虚拟机内部，然后ssh连接进虚拟机，docker run -t是挂载虚拟机内部的目录。
+这里感谢我们的远程助教杉木和Fery提供的解决方案。
```
Comment (lijiarui) [inline _posts/2018-02-06-techie-festival-FAQ.md]: 加上杉木和fery 的github地址：
`[杉木](https://github.com/binsee)` `[Fery](https://github.com/h4dex)`

**Review by lijiarui — CHANGES_REQUESTED** [review]
some small tips, thanks!

**Comment by lijiarui** [comment]
also, please change `2018-02-06-techie-festival-FAQ.md` to `2018-02-06-techie-festival-faq.md`

**Comment by TingYinHelen** [comment]
I have already change the file name.

**Comment by lijiarui** [comment]
Thanks！

**Pull request merged**